apps/bfd-shared-utils/pom.xml

```diff
@@ -16,13 +16,16 @@
 	</description>
 
 	<dependencies>
-			<dependency>
-				<!-- Just useful. -->
-				<groupId>com.google.guava</groupId>
-				<artifactId>guava</artifactId>
-				<version>24.1.1-jre</version>
-			</dependency>
-
+		<dependency>
+			<!-- Just useful. -->
+			<groupId>com.google.guava</groupId>
+			<artifactId>guava</artifactId>
+			<version>24.1.1-jre</version>
+		</dependency>
+		<dependency>
+			<groupId>jakarta.xml.bind</groupId>
+			<artifactId>jakarta.xml.bind-api</artifactId>
+		</dependency>
```

Comment:
Do we still need this? I think either things should be here and therefore automatically picked up by the other modules that depend on this OR they should be in those other modules (which I also see) but not both.

Reply:
Needed for the exception template files in the shared-utils